homeassistant/components/octoprint/__init__.py

```diff
@@ -51,10 +55,28 @@ def ensure_valid_path(value):
     return value
 
 
+@dataclass
+class OctoprintBinarySensorMetadata(BinarySensorEntityDescription):
+    """Metadata for an individual Skybell binary_sensor."""
+
+    endpoint: str = ""
+    group: str = ""
```

Comment:
```suggestion
    endpoint: str
    group: str
```

Reply:
That will not work, because the origin have a parameter with a default value, as a consequence when you inherit you need to set default values.